src/lib/database/entities/GuildEntity.ts

```diff
@@ -66,7 +66,7 @@ export class GuildEntity extends BaseEntity {
 	@Column('varchar', { name: 'disabledCommands', length: 32, array: true, default: () => 'ARRAY[]::VARCHAR[]' })
 	public disabledCommands: string[] = [];
 
-	@ConfigurableKey({ dashboardOnly: true, description: LanguageKeys.Settings.DashboardOnlyKey, type: 'customcommand' })
+	@ConfigurableKey({ dashboardOnly: true, dashboardArray: true, description: LanguageKeys.Settings.DashboardOnlyKey, type: 'customcommand' })
```

Comment:
All you have to do is specify `array: true`.

Reply:
Wrong. That breaks TypeORM.